spec/factories/users.rb

```diff
@@ -109,6 +109,12 @@
       end
     end
 
+    trait :with_pii do
+      after :build do |user|
+        create(:profile, :active, :with_pii, user: user)
+      end
+    end
```

Comment:
Could we reuse the `:proofed` trait here? (or maybe alias/use that trait inside this trait)

Reply:
We ended up removing this because it didn't work. If we need something similar again, we'll take a look at :proofed, thanks!